examples/swarm-setup.go

```diff
@@ -0,0 +1,35 @@
+package main
+
+import (
+	metrics "github.com/ipfs/go-libp2p/p2p/metrics"
+	tu "github.com/ipfs/go-libp2p/testutil"
+	ma "github.com/jbenet/go-multiaddr"
+	swarm "go-libp2p/p2p/net/swarm"
```

Comment:
Shed some light :) I'm trying to get it to work too. I was hoping that I could use gx, but installing globally is hanging https://github.com/whyrusleeping/gx/issues/23